hera_cal/reflections.py

```diff
@@ -309,85 +701,44 @@ def construct_reflection(freqs, amp, tau, phs, real=False):
             input frequencies.
     """
     # make reflection
-    eps = amp * np.exp(2j * np.pi * tau / 1e9 * freqs + 1j * phs)
+    eps = amp * np.exp(2j * np.pi * tau * freqs + 1j * phs)
     if real:
         eps = eps.real
 
     return eps
 
 
-def fit_reflection(data, dly_range, freqs, full_freqs=None, edgecut=0,
-                   taper='none', zero_pad=0, real=False, fthin=10, **win_kwargs):
+def fit_reflection_delay(dfft, dly_range, dlys, return_peak=False):
```

Comment:
It seems like there's a lot of repeated code between this and fft_dly. Can you modify one or both of them to eliminate the repeated code?

Reply:
There is a little bit of repeated code, but its just the peak shifting thats similar: the two functions are working with fundamentally different inputs, so there is no easy way to use one for both.